magefile.go

```diff
@@ -1950,6 +1950,14 @@ func (Integration) TestBeatServerless(ctx context.Context, beatname string) erro
 	return integRunner(ctx, false, "TestBeatsServerless")
 }
 
+func (Integration) TestExtendedRuntime(ctx context.Context) error {
```

Comment:
```suggestion
func (Integration) TestLongRunningAgentForLeaks(ctx context.Context) error {
```

The target name should match what it actually does.